src/components/OrgUpcomingEvents.tsx

```diff
@@ -0,0 +1,40 @@
+import { api } from "@src/utils/api";
+import { type FC } from "react";
+import { SelectClub as Club } from "@src/server/db/models";
+
+const cur_time = new Date();
+
+const OrgUpcomingEvents:FC<{club_id : string}> = ({club_id}) => {
+
+  const datt = new Date("2025, 5, 4")
```

Comment:
Maybe remove this

Reply:
oh good catch! I forgot to remove that. I just removed it